nsv13/code/game/machinery/iff_console.dm

```diff
@@ -33,6 +33,9 @@ If someone hacks it, you can always rebuild it.
 /obj/machinery/computer/iff_console/LateInitialize()
 	. = ..()
 	var/obj/structure/overmap/OM = get_overmap()
+	if(OM.hammerlocked == TRUE) //In case the console gets destroyed and needs to be rebuilt
```

Comment:
For the sake of code safety I recommend making this OM?.hammerlocked in case something stupid happens and it can't find its overmap